packages/actor-init-query/lib/cli/CliArgsHandlerBase.ts

```diff
@@ -123,6 +123,18 @@ export class CliArgsHandlerBase implements ICliArgsHandler {
           type: 'boolean',
           describe: 'Makes the HTTP timeout take into account the response body stream read',
         },
+        httpRetryCount: {
+          type: 'number',
+          describe: 'The number of retries to perform on failed fetch requests',
+        },
+        httpRetryDelay: {
+          type: 'number',
+          describe: 'The number of milliseconds to wait between fetch retries',
+        },
+        httpRetryOn5xx: {
```

Comment:
I'm wondering if this would be better named as `httpRetryOnServerError`. Probably wouldn't bother changing it unless other changes need be made as well.

Reply:
Had this in mind as well, so might be good to change still indeed.

Reply:
Got that changed :+1: